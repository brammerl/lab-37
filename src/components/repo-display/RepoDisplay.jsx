import React from 'react';
import { useRepos } from '../../hooks/githubContext';

const RepoDisplay = () => {
 const repos  = useRepos()
return (
  <ul>
    <h3>Repositories</h3>
    {
      repos.map(repo => <li key={repo.name}>{repo.name}</li>)
    }
  </ul>
)
}

export default RepoDisplay;