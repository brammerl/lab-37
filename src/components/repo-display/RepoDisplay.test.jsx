import React from 'react';
import { shallow } from 'enzyme';
import RepoDisplay from './RepoDisplay';


jest.mock('./RepoDisplay')

describe('RepoDisplay', () => {
  it('renders RepoDisplay', () => {
    const wrapper = shallow(<RepoDisplay />);
    expect(wrapper).toMatchSnapshot();
  });
});
  