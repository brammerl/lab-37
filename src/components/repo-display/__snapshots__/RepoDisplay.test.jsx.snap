// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RepoDisplay renders RepoDisplay 1`] = `""`;
